group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupCoordinatorServiceTest.java

```diff
@@ -1429,14 +1434,14 @@ public void testConsumerGroupDescribe() throws InterruptedException, ExecutionEx
         when(runtime.scheduleReadOperation(
             ArgumentMatchers.eq("consumer-group-describe"),
             ArgumentMatchers.eq(new TopicPartition(Topic.GROUP_METADATA_TOPIC_NAME, 0)),
-            ArgumentMatchers.any()
+            readOperationCaptor.capture()
```

Comment:
Can't we directly verify the expected list here with `ArgumentMatchers.eq`?

Reply:
The argument here is a lambda that we cannot check directly. We need to capture it, run it. Inside the lambda, we're calling the method with the list to be checked. That's the testing complication here.